addons/test_mail_full/tests/test_portal.py

```diff
@@ -162,3 +165,143 @@ def test_portal_mixin(self):
 
         record_portal._portal_ensure_token()
         self.assertTrue(record_portal.access_token)
+
+
+@tagged('portal')
+class TestPortalFlow(MailCommon, HttpCase):
```

Comment:
I would define it before TestPortalMixin as classes are called by alphabetical order (easier when doing refactoring, having tests following the file order, better when having several tests to fix or check :D )

Reply:
Changed.